pkg/backupdriver/backup_driver_controller_base.go

```diff
@@ -289,6 +266,27 @@ func NewBackupDriverController(
 		resyncPeriod,
 	)
 
+	uploadInformer.Informer().AddEventHandlerWithResyncPeriod(
+		cache.ResourceEventHandlerFuncs{
+			//AddFunc:    func(obj interface{}) { ctrl.enqueueUpload(obj) },
+			UpdateFunc: func(oldObj, newObj interface{}) { ctrl.enqueueUpload(oldObj, newObj) },
+			//DeleteFunc: func(obj interface{}) { ctrl.delUpload(obj) },
+		},
+		resyncPeriod,
+	)
+
+	// Configure supervisor cluster informers in the guest
+	if svcKubeConfig != nil && !localMode {
+		svcSnapshotInformer := svcBackupdriverInformerFactory.Backupdriver().V1().Snapshots()
+		svcSnapshotInformer.Informer().AddEventHandlerWithResyncPeriod(
+			cache.ResourceEventHandlerFuncs{
+				//AddFunc:    func(obj interface{}) { ctrl.enqueueSvcSnapshot(obj) },
+				UpdateFunc: func(oldObj, newObj interface{}) { ctrl.enqueueSvcSnapshot(oldObj, newObj) },
```

Comment:
enqueueSvcSnapshot should be renamed to something like updateSvcSnapshot as we can't use the same function for add and update any more.